SzczepionkaAPI/src/main/java/com/szczepionka/service/EmailService.java

```diff
@@ -0,0 +1,117 @@
+package com.szczepionka.service;
+
+import javax.mail.*;
+import javax.mail.internet.*;
+
+import com.szczepionka.entity.Patient;
+import com.szczepionka.model.PatientDTO;
+import com.szczepionka.repository.PatientRepository;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import org.springframework.web.util.UriComponents;
+import org.springframework.web.util.UriComponentsBuilder;
+
+import java.io.IOException;
+import java.util.Date;
+import java.util.Optional;
+import java.util.Properties;
+
+@Service
+public class EmailService {
+
+    private final PatientRepository patientRepository;
+
+    @Value("${gmail.username}")
+    private String username;
+    @Value("${gmail.password}")
+    private String password;
+
+    public EmailService(PatientRepository patientRepository) {
+        this.patientRepository = patientRepository;
+    }
+
+
+    public PatientDTO sendmail(Long patientId) throws  AddressException, MessagingException, IOException {
+        Optional <Patient> patient = patientRepository.findById(patientId);
+        if(patient.isPresent()) {
+            PatientDTO patientDTO = PatientDTO.builder().
+                    email(patient.get().getEmail())
+                    .pesel(patient.get().getPesel())
+                    .postalCode(patient.get().getPostalCode())
+                    .referralId(patient.get().getReferralId())
+                    .build();
+
+            Properties props = new Properties();
```

Comment:
I'm proposing to just name it properties.

Reply:
fixed